### DISCUSSION TITLE
Tab container hover and content-element-separation

### DISCUSSION THREAD

**Discussion by curoviyxru** [zen-browser/desktop#1372] [Ideas]
Hi. If `zen.theme.content-element-separation` is set to zero, tab bar container doesn't open on hover. I've created a theme to mitigate this for myself:
```
@media (-moz-bool-pref: "zen.view.compact") {
  #navigator-toolbox {
    --zen-compact-toolbox-margin-single: calc(10px * 2) !important;
  }
}
```

Is there any chance for it to be improved so hover container will not resize with this property?

**Comment by mr-cheffy** [comment]
Yeah, the compact mode depends on the element separation, otherwise it would overlap the website's content... Im actively trying to find a solution for this
